src/Codeception/Template/Upgrade4.php

```diff
@@ -0,0 +1,66 @@
+<?php
+namespace Codeception\Template;
+
+use Codeception\Configuration;
+use Codeception\InitTemplate;
+
+class Upgrade4 extends InitTemplate
+{
+    const SURVEY_LINK = 'http://bit.ly/codecept-survey';
+    const DONATE_LINK = 'https://opencollective.com/codeception';
+
+    public function setup()
+    {
+        if (!$this->isInstalled()) {
+            $this->sayWarning('Codeception is not installed in this dir.');
+            return;
+        }
+        $this->sayInfo('Welcome to Codeception v4 Upgrade wizard!');
+        $this->say('');
+        $this->say('Codeception is maintained since 2011, is free & open-source.');
+        $this->say('To make it better we need your feedback on it!');
+        $this->say('');
+        $this->say('Please take a minute and fill in a brief survey:');
+        $this->say('<bold>'  . self::SURVEY_LINK . '</bold>');
+        sleep(5);
+        $this->say('');
+        $result = $this->ask('<question>Did you fill in the survey?</question>', true);
+        if ($result) {
+            $this->say('Thank you! ');
+        } else {
+            $this->say('Anyway...');
+        }
+        $config = Configuration::config();
+        $modules = [];
+        $suites = Configuration::suites();
+        if (empty($suites)) {
+            throw new \Exception("No suites found in codeception.yml. Use upgrade script on config with suites");
```

Comment:
This message is confusing.
Suites are almost never configured in codeception.yml, unless it is single suite project where suite config is merged to codeception.yml.

`codecept init upgrade4 -c backend/` works, but it must be documented,